app/assets/stylesheets/zammad.scss

```diff
@@ -8085,8 +8090,8 @@ output {
   }
 
   .dropdown-menu {
-    margin-top: -3px;
```

Comment:
Why were these changes required? Looks like they might break other parts of the application.